app/assets/v2/css/dashboard.css

```diff
@@ -269,3 +269,10 @@ body {
 .btn-expand-token:focus {
   color: #007bff;
 }
+
+.expired-icon {
```

Comment:
with the changes to font I think only the font-size will be enough maybe margin also.

Reply:
Sure! Changed the `font-size` to `15px`, and kept the margin (to seperate the clock icon from the following text). Also kept the transform `translateY(5%);` though. Without it, the clock is off center in comparison to the text.

On another hand, I can up the font-size of the text as well and send an image if you'd like.

Reply:
@Anish-Agnihotri nah this should be alright ^_^

Reply:
Just a minor thing but transformations are heavy tasks for the browser, I believe we should use vertical align or other text focus css properties.

Reply:
Sure, that’s not a problem at all. I’ll jump in and remove the transform and move over to padding or another text focus property. I’ll push another PR in ~2hrs, not at home atm.

Reply:
Sorry `vertical-align: text-bottom;` to be exact the same position of the transform